installation/cmd/run.sh

```diff
@@ -3,34 +3,27 @@
 set -o errexit
 
 ROOT_PATH=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/../..
-
-defaultRelease="latest-published"
+defaultRelease=$(<"${ROOT_PATH}"/installation/resources/KYMA_VERSION)
 KYMA_RELEASE=${1:-$defaultRelease}
 COMPASS_HELM_RELEASE_NAME="compass"
 COMPASS_HELM_RELEASE_NAMESPACE="compass-system"
-
 INSTALLER_CR_PATH="${ROOT_PATH}"/installation/resources/installer-cr-kyma-diet.yaml
 OVERRIDES_COMPASS_GATEWAY="${ROOT_PATH}"/installation/resources/installer-overrides-compass-gateway.yaml
 ISTIO_OVERRIDES="${ROOT_PATH}"/installation/resources/installer-overrides-istio.yaml
+API_GATEWAY_OVERRIDES="${ROOT_PATH}"/installation/resources/installer-overrides-api-gateway.yaml
 MINIKUBE_HELM_VALUES="${ROOT_PATH}"/installation/resources/minikube-values.yaml
 
 kyma provision minikube
-kyma install -o $INSTALLER_CR_PATH  -o $OVERRIDES_COMPASS_GATEWAY -o $ISTIO_OVERRIDES --source "${KYMA_RELEASE}"
+kyma install -o $INSTALLER_CR_PATH  -o $OVERRIDES_COMPASS_GATEWAY -o $ISTIO_OVERRIDES -o $API_GATEWAY_OVERRIDES "${KYMA_RELEASE}"
 
 #Get Tiller tls client certificates
 kubectl get -n kyma-installer secret helm-secret -o jsonpath="{.data['global\.helm\.ca\.crt']}" | base64 --decode > "$(helm home)/ca.pem"
 kubectl get -n kyma-installer secret helm-secret -o jsonpath="{.data['global\.helm\.tls\.crt']}" | base64 --decode > "$(helm home)/cert.pem"
 kubectl get -n kyma-installer secret helm-secret -o jsonpath="{.data['global\.helm\.tls\.key']}" | base64 --decode > "$(helm home)/key.pem"
 echo -e "Secrets with Tiller tls client certificates have been created \n"
 
-MINIKUBE_IP=$(eval minikube ip)
-helm install \
--f=${MINIKUBE_HELM_VALUES} \
---set=global.minikubeIP=${MINIKUBE_IP} \
---name "${COMPASS_HELM_RELEASE_NAME}" \
---namespace "${COMPASS_HELM_RELEASE_NAMESPACE}" \
-"${ROOT_PATH}"/chart/compass \
---tls --wait
+bash "${ROOT_PATH}"/installation/scripts/run-compass-installer.sh
+bash "${ROOT_PATH}"/installation/scripts/is-installed.sh
 
 # TODO: Remove it after next CLI release
```

Comment:
Seems like CLI [adds](https://github.com/kyma-project/cli/blob/47ab12cc176b30dffe8ae2c0abd7e174b5c3c8ad/cmd/kyma/install/cmd.go#L224) to hosts all virtual services now, maybe we could remove this already? :)